### ISSUE TITLE
CMS URL segment localizable

### ISSUE THREAD

**Issue by Futskito** [aimeos/aimeos-laravel#470]
How can i localize the url segment of a CMS page for any language? I couldn't find the way to make it. Is this possible?

**Comment by aimeos** [comment]
You can create CMS pages with different, localized URLs and content in the language for that URL

**Comment by Futskito** [comment]
Yes but if i create one cms page per language, then I lose the canonical urls feature and it can be marked as repetitive content by crawlers

**Comment by aimeos** [comment]
Canonical URLs are for pages with same content and they aren't suitable if you have text in different languages ("hreflang" would be correct in that case: https://moz.com/learn/seo/hreflang-tag). So there's no issue.
